client/galaxy/scripts/mvc/tool-recommendation.js

```diff
@@ -0,0 +1,237 @@
+import Backbone from "backbone";
+import Utils from "utils/utils";
+import * as d3 from "d3";
+import { getAppRoot } from "onload/loadConfig";
+import $ from "jquery";
+
+var ToolRecommendationView = Backbone.View.extend({
```

Comment:
Sorry that I join so late on this topic but is it possible to integrate this as vue component?

Reply:
I will look into it. Thanks!